PROJECT

```diff
@@ -25,4 +25,13 @@ resources:
   kind: FeatureFlagConfiguration
   path: github.com/open-feature/open-feature-operator/apis/core/v1alpha2
   version: v1alpha2
+- api:
+    crdVersion: v1
+    namespaced: true
+  controller: true
+  domain: openfeature.dev
+  group: core
+  kind: FlagdConfiguration
```

Comment:
My 2c but the custom resource should be `OpenFeatureOperatorConfigruation` or `OFOConfiguration` because `flagD` is the current in-vogue provider for injecting into the workloads. Having a CRD for each provider doesn't seem scalable, in addition to that, I would also like to put our more esoteric configuration for OFO, especially as we consider allowing for flagD upgrades in the OFO configuration.

Reply:
I see your point... but we're not necessarily configuring the operator itself with this CRD are we? We are configuring the `source-of-truth` for the flags in the relevant workload(s). 

What about `FlagSource`? shortname `fs` or something like that?

Reply:
+1 for `ffs`

I agree with Todd here that these configurations apply to the workloads so something along the lines on `SidecarConfiguration` or `FlagSourceConfiguration` would be more suitable

Reply:
Hey @AlexsJones, does this address your concerns?

Reply:
I have updated the name of the CRD to `FlagSourceConfiguration` along with its annotation